ecosystem/templates/configured_tox.ini

```diff
@@ -6,6 +6,7 @@ skipsdist = True
 [testenv]
 usedevelop = true
 install_command = pip install -U {opts} {packages}
+passenv = *
```

Comment:
Could this expose our secrets to projects?

Reply:
Nop, only var declare as env variable in the shell who call the tox